packages/flame/lib/src/components/input/hud_button_component.dart

```diff
@@ -58,8 +75,10 @@ class HudButtonComponent extends HudMarginComponent with Tappable {
   @mustCallSuper
   bool onTapCancel() {
     if (buttonDown != null) {
-      children.remove(buttonDown!);
-      add(button);
+      remove(buttonDown!);
+      if (button != null) {
```

Comment:
Isn't this another change?

Reply:
It is streamlining so that you can skip putting things in the constructor and create them in `onLoad` instead, like we do in most components extending `PositionComponent`.